website/source/guides/node-draining.html.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: "guides"
+page_title: "Decommissioning Nodes"
+sidebar_current: "guides-decommissioning-nodes"
+description: |-
+  Decommissioning nodes is a normal part of cluster operations for a variety of
+  reasons: server maintenance, operating system upgrades, etc. Nomad offers a
+  number of parameters for controlling how running jobs are migrated off of
+  draining nodes.
+---
+
+# Decommissioning Nomad Client Nodes
+
+Decommissioning nodes is a normal part of cluster operations for a variety of
+reasons: server maintenance, operating system upgrades, etc. Nomad offers a
+number of parameters for controlling how running jobs are migrated off of
+draining nodes.
+
+## Configuring How Jobs are Migrated
+
+In Nomad 0.8 a [`migrate`][migrate] stanza was added to jobs to allow control over how
+allocations for a job are migrated off of a draining node. For example for a
+job that runs a web service and has a Consul health check:
+
+```hcl
+job "webapp" {
+  datacenters = ["dc1"]
+
+  migrate {
+    max_parallel = 2
+    health_check = "checks"
+    min_healthy_time = "15s"
+    healthy_deadline = "5m"
+  }
+
+  group "webapp" {
+    count = 9
+
+    task "webapp" {
+      driver = "docker"
+      config {
+        image = "hashicorp/http-echo:0.2.3"
+        args  = ["-text", "ok"]
+        port_map {
+          http = 5678
+        }
+      }
+
+      resources {
+        network {
+          mbits = 10
+          port "http" {}
+        }
+      }
+
+      service {
+        name = "webapp"
+        port = "http"
+        check {
+          name = "http-ok"
+          type = "http"
+          path = "/"
+          interval = "10s"
+          timeout  = "2s"
+        }
+      }
+    }
+  }
+}
+```
+
+The above `migrate` stanza ensures only 2 allocations are stopped at a time to
```

Comment:
I would try to call out that the max_parallel takes effect even if many nodes are draining.